R/layers.R

```diff
@@ -943,6 +943,53 @@ addGeoJSON = function(map, geojson, layerId = NULL, group = NULL,
   invokeMethod(map, getMapData(map), 'addGeoJSON', geojson, layerId, group, options)
 }
 
+#' @param style a JSON string
+#' @describeIn map-layers change style of GeoJSON feature
+#' @export
+styleFeatureGeoJSON = function(map, layerId = NULL, featureId = NULL, style = NULL) {
+  if(class(layerId) != "character" || length(layerId) != 1) {
```

Comment:
`class(foo) == 'bar'` is not an appropriate way to test the class in R, since an object may have multiple classes. Use `inherits(foo, 'bar')` if the function `is.bar()` does not exist (in this case, just use `is.character()`).